gitpod-io/website
# Change wording on get self hosted enterprise license page to remove 10 users

## Description
Changes the wording on the [enterprise license](https://www.gitpod.io/enterprise-license) page to not say "get 10 additional users". 10 users specifically is not relevant anymore. Instead it should say get additional users.

## Related Issue(s)

Fixes #1988

## How to test
- Open preview, navigate to pricing > Self Hosted > ´Register and Install Now` on the professional license
- We should also test the functioning of the form, just in case.  --> Tested, this works! 

## Release Notes

```release-note
NONE
```


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2038"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2038"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
